src/server.js

```diff
@@ -18,7 +18,16 @@ import sensorthings     from './routes/sensorthings';
 let app = express();
 
 app.use(bodyParser.urlencoded({ extended: false }));
-app.use(expressValidator());
+app.use(expressValidator({
+  customValidators: {
+    isArrayOfUrls: (value, options) => {
+      const validator = expressValidator.validator;
+      return Array.isArray(value) &&
+             value.length > 0 &&
+             value.every(item => validator.isUrl(item, options));
```

Comment:
I realize that it is isURL we want here. I don't know why we didn't see it earlier, but now this fails badly for me.